shared/js/contacts/import/utilities/sdcard.js

```diff
@@ -139,31 +139,50 @@ if (!utils.sdcard) {
   };
 
   /**
-   * Extracts and concatenates text from the given array of file objects.
+   * Incrementally extracts vcard text from a file.
    *
-   * @param {Array} fileArray Array of File Objects.
-   * @param {String} contents Accumulated text from the previous recursive call.
-   * @param {Function} cb Function to call when the work is finished.
+   * @param {Object} file File Object.
+   * @param {Function} cb Function to call for each chunk of vcards retrieved
+   *                      from the file.
    */
-  SdCard.getTextFromFiles = function(fileArray, contents, cb) {
-    contents = contents || '';
-    if (!fileArray || !fileArray.length) {
-      return cb && cb(null, contents);
-    }
+  SdCard.getTextFromFiles = function(file, cb) {
```

Comment:
We should change the function name to express that now we are not passing an array, but a single file:
`getTextFromFile`